code/__DEFINES/combat.dm

```diff
@@ -49,7 +49,7 @@
 //Actual combat defines
 
 //click cooldowns, in tenths of a second, used for various combat actions
-#define CLICK_CD_DISARM 16 // Disarm actions have a longer cooldown before the next action can be taken
+#define CLICK_CD_DISARM 12 // Disarm actions have a longer cooldown before the next action can be taken
```

Comment:
honestly, I think this should just be a normal click CD

Reply:
Reverted entirely now, but also reduced the stun duration to match so that they can't be chained end to end.